Dockerfile

```diff
@@ -1,48 +1,61 @@
-FROM python:3.9.4-alpine3.13
+FROM alpine:3.13 AS builder
```

Comment:
I ended up rolling a multi-stage build so we could try to take advantage of caching some layers. I'd like to optimize this more and remove the librdkafka "build from source" step we're using. We have to build from source now since the Alpine repos don't support librdkafka 1.16 in a stable package repo yet.